components/studio-platform/features/org.wso2.integrationstudio.platform.dependencies.feature/feature.xml

```diff
@@ -32,41 +32,41 @@
+      <plugin id="org.eclipse.gmf.runtime.diagram.ui.resources.editor.ide" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.diagram.ui.resources.editor" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.diagram.ui" download-size="0" install-size="0" version="1.9.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.draw2d.ui.render.awt" download-size="0" install-size="0" version="1.8.1.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.draw2d.ui.render" download-size="0" install-size="0" version="1.7.1.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.draw2d.ui" download-size="0" install-size="0" version="1.9.1.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.commands.core" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.core" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.type.core" download-size="0" install-size="0" version="1.9.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.ui.properties" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.ui" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.gef.ui" download-size="0" install-size="0" version="1.7.1.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.notation" download-size="0" install-size="0" version="1.10.0.202004160913" unpack="false"/>
+      <plugin id="org.eclipse.ocl.ecore" download-size="0" install-size="0" version="3.17.0.v20211130-1448" unpack="false"/>
+      <plugin id="org.eclipse.ocl" download-size="0" install-size="0" version="3.17.0.v20211130-1448" unpack="false"/>
+      <plugin id="org.eclipse.ocl.common" download-size="0" install-size="0" version="1.17.0.v20211130-1448" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.common.ui.printing.win32" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
+      <plugin id="org.eclipse.gmf.runtime.emf.clipboard.core" download-size="0" install-size="0" version="1.7.0.202106221344" unpack="false"/>
       <plugin id="org.junit4" download-size="0" install-size="0" version="4.8.1.v20120523-1257-wso2v1" unpack="false"/>
-      <plugin id="org.eclipse.papyrus.infra.core.architecture" download-size="0" install-size="0" version="0.0.0" unpack="false"/>
+      <plugin id="org.eclipse.papyrus.infra.core.architecture" download-size="0" install-size="0" version="3.1.0.202112011019" unpack="false"/>
 
-</feature>
+</feature>
```

Comment:
Why have we removed the newline at the end? Was this intentional?

Reply:
Fixing : [1129](https://github.com/wso2/integration-studio/pull/1129)